Patches/FashionRIMsta/ThingsDefs_Misc/FashionRIMsta_Apparel.xml

```diff
@@ -1,163 +1,186 @@
-        </value>
-     </li>
-
-     <li Class="PatchOperationAdd">
-        <xpath>Defs/ThingDef[defName="Apparel_FSGasmask"]/statBases</xpath>
-        <value>
-           <Bulk>5</Bulk>
-           <WornBulk>3</WornBulk>
-        </value>
-     </li>
-
-   </operations>
-  </Operation>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>FashionRIMsta</modName>
+			</li>
+
+			<!-- Coats -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSFurCoat"]/statBases</xpath>
+				<value>
+					<Bulk>12</Bulk>
+					<WornBulk>6</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSFurCoat"]/equippedStatOffsets/MoveSpeed</xpath>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSFurCoat"]/equippedStatOffsets/WorkSpeedGlobal</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSDesertRobe"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSGardeningVest"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>1</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSGardeningVest"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>10</CarryBulk>
+				</value>
+			</li>
+
+			<!-- Pants -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSThermalPants"]/statBases</xpath>
+				<value>
+					<Bulk>3</Bulk>
+					<WornBulk>1</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSCargoShorts"]/equippedStatOffsets/MoveSpeed</xpath>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSCargoShorts"]/equippedStatOffsets/WorkSpeedGlobal</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSCargoShorts"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>5</CarryBulk>
+				</value>
+			</li>
+
+			<!-- Middle -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSOveralls"]/statBases</xpath>
+				<value>
+					<Bulk>4</Bulk>
+					<WornBulk>1</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSOveralls"]/equippedStatOffsets/MoveSpeed</xpath>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[defName="Apparel_FSOveralls"]/equippedStatOffsets/WorkSpeedGlobal</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_FSOveralls"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>10</CarryBulk>
```

Comment:
As basic clothing (shirt, pants, etc.) in CE doesn't add CarryBulk, the clothing items here may be somewhat too high--could perhaps get away with up to maybe 5 at most for especially spacious items?

Reply:
These values were originally defined in 2017 here in CE, and the FT team made no further adjustments since.

That said, I can cap most of these down to `<CarryBulk>5</CarryBulk>`.

Reply:
Fixed in f30462c